### DISCUSSION TITLE
材质和光影的收集或推荐

### DISCUSSION THREAD

**Discussion by liulila** [Meloong-Git/PCL#1914] [讨论]
材质和光影的收集或推荐

**Comment by WForst-Breeze** [comment]
看个人偏好吧……？我个人喜欢优雅原版向的，一般搭配是 **[Stay True](https://www.curseforge.com/minecraft/texture-packs/stay-true)** + **[Complementary Shaders](https://www.curseforge.com/minecraft/customization/complementary-shaders "之所以推荐 Complementary Shaders，是因为其兼容性非常不错，画面表现力也是首屈一指的！")** 🤔
如果加装了 Optifine 或者 EMF Mod，可以试试 **[Fresh Animations](https://www.curseforge.com/minecraft/texture-packs/fresh-animations)**

功能向推荐 Xekr 红显，还有比较作弊的就是 Xray 矿透了

---

同时有一些知名的优秀光影：诸如 **[BSL](https://www.curseforge.com/minecraft/customization/bsl-shaders)**、**[SEUS](https://www.sonicether.com/seus/)**（其中 SEUS PTGI 表现最佳，但也要注意一下自己的电脑配置……）、**[Sildur's Vibrant shaders](https://www.curseforge.com/minecraft/customization/sildurs-vibrant-shaders)**、**[Complementary Reimagined](https://www.curseforge.com/minecraft/customization/complementary-reimagined)** 以及 **[Kappa](https://www.curseforge.com/minecraft/customization/kappa-shader-by-rre36)**

**Reply by liulila** [reply]
没想到这么快就有人回复 蟹蟹你的推荐我也喜欢这个材质和光影

**Reply by WForst-Breeze** [reply]
> 没想到这么快就有人回复 蟹蟹你的推荐我也喜欢这个材质和光影

举手之劳~ 其实还有不少优秀的光影和资源包，也不一一列举了，你可以参考下面几个链接~ 网站都是英文的而且可能比较慢，如果还需要啥帮助尽管告诉我🥳

光影包：[【CurseForge】](https://www.curseforge.com/minecraft/search?page=1&class=customization&sortType=1&pageSize=20)[【Modrinth】](https://modrinth.com/shaders)

资源包：[【Curseforge】](https://www.curseforge.com/minecraft/search?page=1&class=texture-packs&sortType=1&pageSize=20)[【Modrinth】](https://modrinth.com/resourcepacks)

**Reply by liulila** [reply]
那我确实有个问题紫色隐藏主题需要下载PCL2 Dev 0.4.4我找不到在哪可以下载🤔麻烦了🌺

**Reply by WForst-Breeze** [reply]
https://afdian.net/p/dba42c22c93c11e8a70452540025c377

**Reply by liulila** [reply]
蟹蟹(●'◡'●)

**Reply by so1ve** [reply]
PTGI YYDS!

**Comment by Chiloven945** [comment]
光影 Complementary 和重构版本，BSL，Superduper都在用
材质的话一般用xk的方纹

**Reply by WForst-Breeze** [reply]
！！！[XeKr 的方纹](https://www.curseforge.com/minecraft/texture-packs/xekr-square-pattern)感觉配色很舒服！我也超喜欢🧐！

**Reply by Chiloven945** [reply]
> !!![XeKr 的方纹](https://www.curseforge.com/minecraft/texture-packs/xekr-square-pattern)感觉配色很舒服！我也超喜欢🧐！

xk基本上会在快照发出来的第一时间就把材质做了，适配挺快的

**Comment by iPodtouch6557** [comment]
光影低配电脑最适合的就是chocapic，但是最高也大概lite比较流畅
我个人不喜欢材质优化，不予推荐

**Comment by peng876** [comment]
墙裂推荐两个宝藏材质smoube和quadral
前者是可爱的原版风格8x材质包 调色好到无须开光影 后者是鲜艳的中世纪风格（？）材质包 细节十分饱满丰富
下载的话在planetminecraft上搜就行
